spec/features/cancelling_a_per_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'feature_helper'
+
+RSpec.feature 'cancelling a PER', type: :feature do
+  scenario 'Cancelling a non issued PER' do
```

Comment:
This covers only the happy scenario. Should we have a `request` spec for the cancelling action?